app/models/reporting/workload_report.rb

```diff
@@ -0,0 +1,60 @@
+module Reporting
+  class WorkloadReport
+    def initialize
+      @number_of_rows = 4
+      @slug = 'workload_report'
+    end
+
+    def id
+      @slug
+    end
+
+    attr_reader :number_of_rows, :to_param
+
+    def headers
+      table.keys.map do |header|
+        I18n.t(header, scope: :table_headings)
+      end
+    end
+
+    def rows
+      table.values.transpose
+    end
+
+    private
+
+    def table
+      {
+        days_passed:,
+        open_applications_by_age:,
+        closed_applications_by_age:
+      }
+    end
+
+    def open_applications_by_age
```

Comment:
I think this would be easier to parse if we had a filter variable to pass in to the contructor:

```ruby
     filter = ApplicationSearchFilter.new(application_status: 'open')
     daily_counts = DailyCount.new(filter: filter)).counts

     daily_counts.map do |content|
        Cell.new(content)
      end
```

Reply:
👍